docs/CONFIG-VARS.md

```diff
@@ -102,15 +108,15 @@ Viya4-deploy uses the jump server to interact with the RWX filestore, which must
 
 ### Azure
 
-When V4_CFG_MANAGE_STORAGE is set to `true`, a new storage class is created: sas (Azure NetApp or NFS).
+When V4_CFG_MANAGE_STORAGE is set to `true`, the `sas` and `pg-storage` storage classes are created (Azure NetApp or NFS).
```

Comment:
```suggestion
When V4_CFG_MANAGE_STORAGE is set to `true`, the `sas` and `pg-storage` storage classes are created by the nfs-subdir-external-provisioner (Azure NetApp or NFS).
```

Reply:
Why are we identifying the tool set being used here? Not sure the advantage. It's now a maintenance item that would have to be checked and/or verified if we ever change. Just asking.

Reply:
Wanted to align Azure with AWS and GCP sections, I've removed that reference as I misunderstood your earlier comment.